occurrences/schema.py

```diff
@@ -406,8 +406,8 @@ def validate_enrolment(study_group, occurrence, new_enrolment=True):
     if occurrence.cancelled:
         raise EnrolCancelledOccurrenceError("Cannot enrol cancelled occurrence")
     if (
-        study_group.group_size > occurrence.max_group_size
-        or study_group.group_size < occurrence.min_group_size
+        study_group.group_size > occurrence.max_group_size > 0
+        or 0 < study_group.group_size < occurrence.min_group_size
```

Comment:
Hmm, doesn't this now throw when `max_group_size` or `min_group_size` is `None` ?

Reply:
Ahhh, my bad, must be to hurry for the lunch, reverted